core/src/storage/impls/state.rs

```diff
@@ -546,6 +546,8 @@ impl<'a> State<'a> {
 
         inserter.iterate(dumper)
     }
+
+    pub fn get_delta_trie(&self) -> Arc<DeltaMpt> { self.delta_trie.clone() }
```

Comment:
Maybe you can use 
    pub fn new_or_get_delta_mpt(
        storage_manager: Arc<StorageManager>, snapshot_epoch_id: &EpochId,
    ) -> Result<Arc<DeltaMpt>>

Reply:
Can you mark this method test only?